src/components/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -101,28 +101,28 @@ const Input = styled(RadioGroup.Item)`
 
   box-sizing: border-box;
   padding: 0;
-  border: 1px solid var(--radio-fg);
+  border: 1px solid var(--echoes-color-background-accent-default);
   border-radius: 100%;
 
-  height: var(--size-dimension-md);
-  width: var(--size-dimension-md);
-  min-width: var(--size-dimension-md);
+  height: var(--echoes-radio-button-group-size-input);
```

Comment:
Here we might want to have `radiobuttongroup` instead of `radio-button-group` what do you think?